R/time_series.R

```diff
@@ -1,5 +1,6 @@
 get_programme_time_series <- function(programme, shape) {
-  data <- read_csv(system_file("extdata/dummy_input_time_series_programme.csv"))
+  data <- naomi::prepare_input_time_series_art(programme$path, shape$path)
+  data <- as.data.frame(data)
```

Comment:
```suggestion
  data <- as.data.frame(data, stringsAsFactors = FALSE)
```

(and elsewhere) not a huge deal, but worth adding if we don't depend on R >= 4.0